### PR TITLE
Add more instances for hosted sombras

### PR DESCRIPTION
**Pull request by giacaglia**
This will help connect the patreon org on dev/staging/prod to a server webhook. Adding new instances for patreon single-tenant sombra instances

## Related Issues

- _[none]_

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

File: render.yaml [hunk 1]
```diff
@@ -17,6 +26,15 @@ services:
     startCommand: cd javascript && yarn && yarn start
```
Comment (michaelfarrell76) [inline render.yaml:26]: how come these are named "typescript" but then run in the javascript folder? we're going to eventually want to run both so we should standardize the names

File: render.yaml [hunk 2]
```diff
@@ -25,3 +43,12 @@ services:
     buildCommand: cd javascript && yarn
     startCommand: cd javascript && yarn && yarn start
     healthCheckPath: /health
+
+  # A webhook service for production - Patreon organization
+  - type: web
+    name: typescript-webhook-prod-patreon
```
Comment (michaelfarrell76) [inline render.yaml:49]: this testing org only exists in dev/staging

**Review by michaelfarrell76 — APPROVED** [review]
dont love having customer names in this repo, but think we should just do it sparingly, move forward for now and maybe clean up the names of our testing orgs later on in this project

File: render.yaml (hunk 1 above)
Comment (giacaglia) [reply render.yaml:26]: ugh. Should be `javascript`

**Pull request merged**